utils/mixins.py

```diff
@@ -46,29 +46,29 @@ def get_context_data(self, **kwargs: Any) -> dict:
 
         context_data = super().get_context_data(**kwargs)
 
-        from_date = None
-        to_date = None
-
-        if "year" in self.kwargs and "month" in self.kwargs:
-            year = int(self.kwargs["year"])
-            month = int(self.kwargs["month"])
-            current_date = timezone.now().date()
-            from_date = timezone.datetime(year=year, month=month, day=1).date()
-            if current_date.year == year and current_date.month == month:
-                to_date = current_date
-            else:
-                to_date = self._set_last_day_of_month_on_date(year=year, month=month)
-
-        if self.model == CustomUser:
-            user = self.object
-        else:
-            user = self.request.user
+        report_set = self.object.report_set.all() if self.model == CustomUser else self.get_queryset()
```

Comment:
Instead of checking `self.model == CustomUser` you can check it this way -> `isinstance(self.model, CustomUser)`

Reply:
That's not an instance: it's **_the_** class.
Changed `==` operator to `is`.

Reply:
Done.